.github/workflows/percy.yml

```
@@ -5,13 +5,19 @@ on:
       - master
       - main
-  pull_request:
+  pull_request_target:
     branches:
       - master
       - main
 
 jobs:
   percy-tests:
+    
+    if: >
```

actionlint (1.7.4)

if: condition "${{ github.event_name == 'push' \n   || (github.event_name == 'pull_request_target' \n       && contains(toJson(github.event.pull_request.labels), 'check-visual-regression')) }}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)